### DISCUSSION TITLE
Request for Public Design Drafts and Quick Search Popup Issue

### DISCUSSION THREAD

**Discussion by TinsFox** [toeverything/AFFiNE#4343] [General]
🤔 Hi there, team AFFiNE! 👋

I was wondering if there are any plans 📆 to make the design draft 📝 for AFFiNE public? I've come across what I believe might be some design-related issues, although I'm not entirely sure.

In commit hash 0b1ba6bf43438f3d5b990e662ed06851ee91d215, I've noticed an unusual problem with the quick search popup. The close button in the top right corner doesn't seem to blend well with the shortcut key prompt. Here's an image for reference:
<img width="700" alt="image" src="https://github.com/toeverything/AFFiNE/assets/33956589/ce9a114f-27bc-47af-8e8a-20fe7984fae5">

As there are no publicly available design drafts at the moment, I'm uncertain whether this is an intentional design choice or if it was overlooked during development.

If this isn't part of the intended design, do you have any plans to make the design resources public? This would be beneficial as it would allow external contributors to make improvements based on the design drafts if similar issues arise in the future. 🌟"

**Comment by houjoe0829** [comment]
Please take a look @xell
